src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamActor.scala

```diff
@@ -21,7 +21,7 @@ trait HttpEventStreamHandle {
 }
 
 class HttpEventStreamActorMetrics() {
-  val numberOfStreams: SettableGauge = Metrics.atomicGauge(ApiMetric, getClass, "number-of-streams")
+  val numberOfStreams: SettableGauge = Metrics.atomicGauge("marathon.http.event.streams")
```

Comment:
seems like count or total should be here.. or active.count `marathon.http.event.streams.active.count`

Reply:
It is a gauge, so `.total` shouldn't be there. I will append `.active`. As for `.count`, please refer to my comment above.